recipes/RAG/RAG_over_NH_Caselaw.ipynb

```diff
@@ -97,9 +97,10 @@
    "outputs": [],
    "source": [
     "!pip install git+https://github.com/ibm-granite-community/granite-kitchen \\\n",
-    "    datasets \\\n",
     "    langchain-huggingface \\\n",
-    "    langchain-milvus"
+    "    langchain-milvus \\\n",
```

Comment:
Let's put this change in a separate commmit

Reply:
Agreed